manage.py

```diff
@@ -29,42 +29,65 @@ def seed_categories():
 
     food = Category.get_by_name("Food")
     if not food:
-        food = Category("Food", "", 0)
+        food = Category()
+        food.name = "Food",
+        food.description = ""
+        food.sort_order = 0
```

Comment:
Ok, Category __init__ was removed at some point and this was never updated to reflect that. I will add it back.